src/dialogs/voice-assistant-setup/cloud/cloud-step-signin.ts

```diff
@@ -139,6 +143,20 @@ export class CloudStepSignin extends LitElement {
           }
         }
 
+        if (errCode === "alreadyconnectederror") {
+          showloudAlreadyConnectedDialog(this, {
```

Comment:
rename to showCloudAlreadyConnectedDialog

Reply:
You don't want it to be loud? 😄